COMETwebapp/Components/ModelDashboard/UnusedElements.razor

```diff
@@ -0,0 +1,93 @@
+    modify it under the terms of the GNU Affero General Public
+    License as published by the Free Software Foundation; either
+    version 3 of the License, or (at your option) any later version.
+
+    The COMET WEB Community Edition is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+    Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see http://www.gnu.org/licenses/.
+------------------------------------------------------------------------------->
+
+@using DevExpress.Blazor
+@inject NavigationManager NavigationManager
+@inject ISessionAnchor ISessionAnchor
+
+<div class="chart">
+    <h3>Unused Elements</h3>
+    <DxChart Data="@Data"  CustomizeSeriesPoint="@PreparePointLabel">
+        <DxChartFullStackedBarSeries Color="System.Drawing.Color.LightSalmon" Name="Used Elements" Filter="@((ElementDefinition e) => !this.UnUsedElements.Contains(e))"
+                                    ArgumentField="@(e => e.Owner.ShortName)" ValueField="@(e => 1)"
+                                    AggregationMethod="Enumerable.Sum">
+            <DxChartSeriesLabel Position="RelativePosition.Inside"/>
+        </DxChartFullStackedBarSeries>
+        <DxChartFullStackedBarSeries Color="System.Drawing.Color.PeachPuff" Name="Unused Elements" Filter="@((ElementDefinition e) => this.UnUsedElements.Contains(e))"
+                                            ArgumentField="@(e => e.Owner.ShortName)" ValueField="@(e => 1)"
+                                            AggregationMethod="Enumerable.Sum">
+            <DxChartSeriesLabel Position="RelativePosition.Inside" />
+        </DxChartFullStackedBarSeries>
+        <DxChartTooltip Enabled="true" Position="RelativePosition.Inside">
+                    <div style="margin: 0.1rem">
+                        <button class="card-btn" @onclick="@(() => AccessData(context.Point.SeriesName, context.Point.Argument))">
+                            More
+                        </button>
+                    </div>
+        </DxChartTooltip>
+        <DxChartLegend Visible="true" Position="RelativePosition.Outside" VerticalAlignment="VerticalEdge.Bottom"/>
+    </DxChart>
+</div>
+
+@code {
+    /// <summary>
+    /// Data in the graph
+    /// </summary>
+    [Parameter]
+    public List<ElementDefinition> Data { get; set; }
+
+    /// <summary>
+    /// List of unused element defintion from the opened definition
+    /// </summary>
+    private List<ElementDefinition> UnUsedElements { get; set; }
+
+    /// <summary>
+    /// Initialize the UnUsedElements list
+    /// </summary>
+    protected override void OnInitialized()
+    {
+        IterationService iterationService = new IterationService(ISessionAnchor.OpenIteration);
```

Comment:
convention is to use the `var` keyword, so please replace with:

`var iterationService = new IterationService(ISessionAnchor.OpenIteration);`